model_analyzer/constants.py

```diff
@@ -27,3 +27,6 @@
 
 # Reports
 TOP_MODELS_REPORT_KEY = "Best Configs Across All Models"
+
+# State Management
+MAX_NUMBER_OF_INTERRUPTS = 3
```

Comment:
Looks like you have a constants file already. Just add `MAX_RETRIES = 1000` here

Reply:
Will fix this.